### ISSUE TITLE
Typing an email address to an input creates a broken test

### ISSUE THREAD

**Issue by arggh** [qawolf/qawolf#490]
I just started toying around with QA Wolf (_awesome idea by the way - when I first learned about Cypress ages ago I thought & hoped it could do what QA Wolf does_)  and noticed that typing an email address to an input creates a broken test, at least when using a swedish keyboard layout on Mac OS where <kbd>@</kbd> is inserted with <kbd>alt</kbd> + <kbd>2</kbd>.

Test output:

```
it('can type into "signupEmail" input', async () => {
    await browser.type(selectors[4], "↓KeyJ↓KeyO↑KeyJ↑KeyO↓KeyH↑KeyH↓KeyN↑KeyN↓AltLeft↓Digit2↑Digit2↑AltLeft↓KeyM↑KeyM↓KeyA↑KeyA↓KeyI↑KeyI↓KeyL↑KeyL↓KeyN↑KeyN↓Backspace↑Backspace↓KeyI↑KeyI↓KeyN↓KeyA↑KeyN↓KeyT↑KeyA↑KeyT↓KeyO↑KeyO↓KeyR↑KeyR↓Period↑Period↓KeyC↑KeyC↓KeyO↓KeyM↑KeyO↑KeyM");
  });
```

...which fails to write the <kbd>@</kbd> character, when trying to run the test.

When replaced with...

```
it('can type into "signupEmail" input', async () => {
    await browser.type(selectors[4], "john@mailinator.com");
  });
```

...the test runs smoohtly.

**Comment by jperl** [comment]
Thanks for reporting this. We are reworking the way typing works in the next release which should fix your issue.

Relates to #441

**Comment by jperl** [comment]
Resolved in #491
